### PR TITLE
Added sync plan enable and disable feature in sync_capsule_repos_to_upgrade.

### PR DESCRIPTION
**Pull request by devendra104**
Updated sync plan enable and disable the feature in the sync_capsule_repos_to_upgrade method.

### CODE REVIEW

File: upgrade/helpers/tasks.py [hunk 1]
```diff
@@ -95,6 +95,8 @@ def sync_capsule_repos_to_upgrade(capsules):
     RHEV_CAPSULE_AK
         The AK name used in capsule subscription
     """
+    logger.info('Disabling the sync plan ...')
+    run('foreman-maintain advanced procedure run sync-plans-disable')
```
Comment (jyejare) [inline upgrade/helpers/tasks.py]: We shouldnt do this - as we may miss issues related to sync plan migrations while enabled. Any special reason for doing this ?

Comment (ntkathole) [reply upgrade/helpers/tasks.py]: @jyejare https://bugzilla.redhat.com/show_bug.cgi?id=1721055#c7

Comment (jyejare) [reply upgrade/helpers/tasks.py]: Ok, then we should bz_bug_is_open here.

**Comment by devendra104** [comment]
This issue was taken care in #328

**Pull request closed without merging**